controller/api_impl/router_api_model.go

```diff
@@ -55,9 +55,10 @@ func MapCreateRouterToModel(router *rest_model.RouterCreate) *network.Router {
 			Id:   stringz.OrEmpty(router.ID),
 			Tags: TagsOrDefault(router.Tags),
 		},
-		Name:        stringz.OrEmpty(router.Name),
-		Fingerprint: router.Fingerprint,
-		Cost:        uint16(Int64OrDefault(router.Cost)),
+		Name:           stringz.OrEmpty(router.Name),
+		Fingerprint:    router.Fingerprint,
+		Cost:           uint16(Int64OrDefault(router.Cost)),
```

Comment:
Missed this on an earlier pass. I think AllowTraversal should default to true (or reverse the flag to NoTraversal). Otherwise we breaks backwards compatibility. This will likely affect the edge changes as well.